src/ts/erroreventutil.ts

```diff
@@ -0,0 +1,61 @@
+import {ErrorMessageMap} from './components/errormessageoverlay';
+
+export namespace ErrorEventUtil {
+  export const defaultErrorMessagesMapping: ErrorMessageMap = {
+    1001: 'The player API is not available after a call to PlayerAPI.destroy.',
+    2010: 'The certificate is not valid',
+    2002: 'License request failed',
+    2013: 'DRM error for the Flash renderer',
+    2008: 'Key error',
+    2003: 'Key or KeyId is missing',
+    2006: 'Unable to create or initialize key session',
+    2004: 'Key size is not supported',
+    2009: 'The key system is not supported',
+    2007: 'The MediaKey object could not be created/initialized',
+    2000: 'Required DRM configuration is missing',
+    2005: 'Unable to instantiate a key system supporting the required combinations',
+    2001: 'The licensing server URL is missing',
+    2011: 'Invalid header key/value pair for PlayReady license request',
+    2012: 'Content cannot be played back because the output is restricted on this machine',
+    3100: 'An Advertising module error has occurred. Refer to the attached AdvertisingError.',
+    3003: 'A module cannot be loaded because one or more dependencies are missing.',
+    3000: 'The definition of the module is invalid (e.g. incomplete).',
+    3001: 'The module definition specifies dependencies but the module is not provided via a function for deferred loading.',
+    3002: 'A module cannot be loaded because it has not been added to the player core.',
+    1400: 'Network error while downloading',
+    1404: 'The Certificate could not be loaded',
+    1401: 'The manifest download timed out',
+    1403: 'The progressive stream download timed out',
+    1402: 'The segment download timed out',
+    1301: 'General error if Flash renderer has an error',
+    1302: 'Flash doesn\'t have sufficient resources',
+    1303: 'The transmuxer could not be initialized',
+    1300: 'Video decoder or demuxer had an error with the content',
+    1112: 'Flash container API not available',
+    1110: 'Native Flash is not authorized by a valid Adobe token',
+    1111: 'Flash doesn\'t have sufficient resources',
```

Comment:
Same text as 1302?

Reply:
Texts are extracted from the [docs](http://cdn.bitmovin.com/player/web/staging/8/docs/enums/errors.playererrorcode.html)